backend/modules/simulation/handlers/chat_handler.py

```
@@ -985,6 +985,16 @@ async def handle_stream_message(
             
             if timeout_result is not None:
+                # Clean up sandbox if simulation completed via timeout
+                try:
+                    timeout_data = json.loads(timeout_result)
+                    if timeout_data.get("simulation_complete") and user_progress.sandbox_id:
+                        from common.services.sandbox_service import sandbox_service
+                        await sandbox_service.delete_sandbox(user_progress.sandbox_id)
+                        user_progress.sandbox_id = None
+                        self.db.commit()
+                except Exception:
+                    pass  # Best-effort cleanup; Daytona auto-deletes after 24h
```

Ruff (0.15.4)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)